src/main/java/TestDB.java

```diff
@@ -0,0 +1,59 @@
+import java.io.*;
+
+import javax.servlet.http.*;
+import javax.servlet.*;
+
+import java.sql.*;
+
+// This servlet tests the connection with the database
+//
+// Outputs
+// - OK if the connection is successful
+// - ERR otherwise, with the error message
+public class TestDB extends HttpServlet {
+
+  public void doGet (HttpServletRequest req,
+                     HttpServletResponse res)
+    throws ServletException, IOException
+  {
+    
+    try {
+
+        // Loading the driver
+        Class.forName("org.apache.derby.jdbc.ClientDriver");
+    }
+    catch(ClassNotFoundException e) {
+
+        PrintWriter out = res.getWriter();
```

Comment:
This line, together with line 42 and 54 can be put once at the start of the doGET method